src/main/java/org/pih/petl/job/config/ExecutionConfig.java

```diff
@@ -0,0 +1,70 @@
+package org.pih.petl.job.config;
+
+import java.sql.Time;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Configures how execution threading, concurrency, and retries on error should be handled
+ * By default, an ExecutionConfig will specify no retries, and one concurrent job,
+ * but this can be adjusted to support executing jobs in parallel, and configuring the number of times a job
+ * execution should be attempted (i.e. to allow for retrying due to connectivity problems)
+ */
```

Comment:
This is just a convenient way to read in this configuration and pass it around from jobs that support it (job-pipeline, iterating-job)